front/app/modules/commercial/insights/admin/containers/Insights/Details/Network/index.tsx

```diff
@@ -314,6 +318,32 @@ const Network = ({
 
   return (
     <Box ref={containerRef} h="100%" position="relative" overflow="hidden">
+      <Box mt="24px" ml="24px" position="absolute" zIndex="1000">
+        <SectionTitle>
+          {formatMessage(messages.networkTitle)}
+          <IconTooltip
+            ml="8px"
+            content={
+              <TooltipContent>
+                <FormattedMessage
+                  {...messages.networkTitleTooltip}
+                  values={{
+                    link: (
+                      <a
+                        href="https://support.citizenlab.co/en/articles/5525933-creating-insights-to-understand-what-participants-are-talking-about"
```

Comment:
I think you'd want a separate message for the link, for internationalization and being able to change the link with minimal dev time (through crowding flow)

Reply:
Indeed, thanks for spotting this, makes perfect sense. Also changed it for another link we had 👍